docs/source/installation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Setup and options
+---
+<h2 id="installation">Installation</h2>
+
+To get started with Apollo Link, install the `apollo-link` npm package. This exports everything you need to get started, though there may already be a link with the functionality you wan't on npm.
```

Comment:
> though there may already be a link with the functionality you wan't on npm

I don't think this is super helpful. We should have one canonical way to get started.